output/filewriter_test.go

```diff
@@ -35,7 +53,7 @@ func TestFileWriterConfig(t *testing.T) {
 		// error cases
 		{
```

Comment:
yes, good point. Also I think I should fix the case I'm talking about in the previous comment:
""{{.Hello}}"  is a valid template, so not caught at config time, but invalid for our use case since, once executed this gives the `/<no value>` string, which is not a valid path.

Reply:
I've added a test case with an malformed pattern.
However, i have a bit of thought to trying to assert that the generated path will be valid, but it's futile. Valid path is an OS-dependent thing. Moreover, if `{{.Field0}}` is in the string, then we can't possibly know that generated path are always going to be valid. 
I've improved the help string for Field0 with these remarks, plus one about the cardinality if the field. In other words,  is the user uses Field0, with great powers comes great responsibility.